linux_os/guide/system/software/updating/security_patches_up_to_date.rule

```diff
@@ -1,7 +1,5 @@
 documentation_complete: true
 
-prodtype: rhel6,rhel7,ubuntu1404,ubuntu1604,ol7
-
```

Comment:
This should keep its `prodtype`. This rule is about updating packages based on a CVE feed, but not all platforms publish CVE feeds.